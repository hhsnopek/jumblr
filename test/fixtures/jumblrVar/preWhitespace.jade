include ../../../lib/jumblr

p
  +jumblrVar('Scallywag')
  | : Oh
  +jumblrVar('Caption', true)
    |  my Captain
