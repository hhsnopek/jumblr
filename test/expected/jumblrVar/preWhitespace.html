<p>{Scallywag}: Oh {Caption} my Captain</p>
